doc/source/release-notes/Percona-XtraDB-Cluster-5.7.12-rc1-26.16.rst

```diff
@@ -0,0 +1,97 @@
+.. rn:: 5.7.12-rc1-26.16
+
+=========================================
+|Percona XtraDB Cluster| 5.7.12-rc1-26.16 
+=========================================
+
+Percona is glad to announce the release of
+|PXC| 5.7.12-rc1-26.16 on Audust 8, 2016.
+Binaries are available from the
+`downloads area
+<http://www.percona.com/downloads/Percona-XtraDB-Cluster-57/LATEST>`_
+or from our :ref:`software repositories <install>`.
+
+|PXC| 5.7.12-rc1-26.16 is based on the following:
+
+* `Percona Server 5.7.12 <http://www.percona.com/doc/percona-server/5.7/release-notes/Percona-Server-5.7.12.html>`_
+
+* `Galera Replicator 3.16 <https://github.com/percona/galera/tree/rel-3.16>`_
+
+New Features
+============
+
+* [PXC-602] **PXC Strict Mode**:
+  Use the :variable:`pxc_strict_mode` variable in the configuration file
+  or the ``--pxc-strict-mode`` option during ``mysqld`` startup.
+  For more information, see :ref:`pxc-strict-mode`.
+
+* [PXC-626] **Galera instruments exposed in Performance Schema**:
+  This includes mutexes, condition variables, file instances, and thread.
+
+Bug Fixes
+=========
+
+* [PXC-233] Fixed error messages.
```

Comment:
Please re-check the PXC issue number.

https://jira.percona.com/browse/PXC-233 (Discuss hanging issues with Philllip)
Wasn't part of this release. It is already closed too.